pkg/resourcemanager/eventhubs/namespace.go

```diff
@@ -64,6 +64,14 @@ func (m *azureEventHubNamespaceManager) DeleteNamespace(ctx context.Context, res
 		resourceGroupName,
 		namespaceName)
 
+    if err != nil {
```

Comment:
Is this just a drive-by?

Reply:
Failed here in CI…

Reply:
Singe you asked offline, this looks "ok" to me (in that it's just doing what is also done above -- obviously the actual pattern is awkward but that's a problem for another day).